rpc/Cargo.toml

```diff
@@ -16,7 +16,7 @@ name = "lnp_rpc"
 [dependencies]
 amplify = "3.13.0"
 strict_encoding = "0.9.0-rc.2"
-bitcoin_scripts = "0.9.0-rc.1"
+bitcoin_scripts = "0.9.0"
```

Comment:
Can we please update the rest of dependencies with `-rc1` removed? They all got released last week, except `lnp2p`/`lnp-core`, which I will release today

Reply:
I bump the libraries to v0.9 release.